### PR TITLE
Decode a Morse code messag (PR)

### PR DESCRIPTION
**Pull request by Doheera-kosi**
# Milestone reached:
## Objectives
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Use Ruby to solve basic coding challenges.

## General requirements
- No linter errors
- Used correct Gitflow.
- Documented work [in a professional way

## Ruby requirements
- Follow our list of best practices for Ruby

## Project requirements
- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase.

- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. 

- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Doheera-kosi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid2

You have submitted a pull request with failing linter checks. Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: decoder.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    return key if character == value
+  end
+end
+
+def decode_word(word)
+  split_word = word.split
+  split_word.each do |i|
+    decode_char(i)
+  end
+end
```
Comment (alexander16108) [inline decoder.rb]: Hi There 👋🏾 ,
Currently, your  decode word function does not return the correct option and returns back the morse code inputted
image

**Review by alexander16108 — CHANGES_REQUESTED** [review]

File: decoder.rb (hunk 1 above)
Comment (Doheera-kosi) [reply decoder.rb]: Hello Alexander,
Thank you for the constructive feedback on our team project.
Issues resolve now.

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**